src/Tools/MicroBuild/copy-symbols.ps1

```diff
@@ -0,0 +1,35 @@
+# Copy our symbols to the Symbols directory.  This simplifies the logic of our achiving 
+# and indexing steps.  
+Param(
+    [string]$binariesPath = $null
+)
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+try
+{
+    $symbolsPath = join-path $binariesPath "Symbols"
```

Comment:
@jasonmalinowski 

> In order to actually send this stuff downstream we have to have it in "symbol store" form

Where is this form described?  When we were chatting over email my understanding was to put it in a flat folder so we could remove our complex matching code.  If there is a more complex form I'll need some docs.